Rakefile

```diff
@@ -2,7 +2,7 @@ require 'bundler/gem_tasks'
 require 'rake/testtask'
 
 Rake::TestTask.new('test') do |t|
-  t.ruby_opts << '-rubygems'
+  t.ruby_opts << '-r rubygems'
```

Comment:
unrelated change>

Reply:
CI was complaining about this (and was in fact preventing me from testing locally)